tests/general/SpaceConsistencyBearTest.py

```diff
@@ -2,6 +2,8 @@
 
 from bears.general.SpaceConsistencyBear import (
     SpaceConsistencyBear, SpacingHelper)
+from coalib.bearlib.aspects import get as get_aspect
```

Comment:
Just `import coalib.bearlib.aspects as coala_aspects`. The module object is a bit magical ;) It has a `__getitem__`. So you can then for example just write `coala_aspects['Indentation']` to get the `Indentation` aspectclass. That way you also get a clear `AspectNotFoundError` when you accidentally mistype the aspectclass name. `.get()` just returns `None` in that case, which makes it harder to track down the actual problem with failing things like `get('Indentayshon')('Python', ...)`

But actually, you could also just directly import the `Indentation` aspectclass here, since it's the only one you are using below...